README.md

```diff
@@ -69,3 +69,55 @@ vault.get_kv_secret(mount_point="example", path="path/test")
    }
 ]
 ```
+
+## Create or Update KV Secret
+Method to create an secret or update an existing one in a given path. 
+
+```python
+vault = VaultProvider()
+secret = {"example": "Test", "number": 42}
+vault.create_or_update_kv_secret(mount_point="example", path="path/test", secret=secret)
+```
+```
+{
+   'request_id': '4ce62ee7-0f88-3efc-d745-5e2fbc423789',
+   'lease_id': '',
+   'renewable': False,
+   'lease_duration': 0,
+   'data': {
+      'created_time': '2020-09-10T00:25:40.92411625Z',
+      'deletion_time': '',
+      'destroyed': False,
+      'version': 1
+   },
+   'wrap_info': None,
+   'warnings': None,
+   'auth': None
+}
+```
+
+## Patch KV Secret
+Method to update an existing path. Either to add a new key/value to the secret and/or update the value for an existing key. Raises an `hvac.exceptions.InvalidRequest` if the path hasn’t been written to previously.
+
+```python
+vault = VaultProvider()
+secret = {"example": "New Test"}
+vault.patch_kv_secret(mount_point="example", path="path/test", secret=secret)
+```
+```
+{
+   'request_id': '7bf2a869-dc66-efa2-3679-814ef76fb447',
+   'lease_id': '',
+   'renewable': False,
+   'lease_duration': 0,
+   'data': {
```

Comment:
Ditto.
If this returns the data (plus the metadata) it would be nice to show how the patch method behaves: changing only the "example" key in this demonstration.

Reply:
It does not return data, I believe that the ideal was for this key to be "metadata", but the SDK uses "data". 

This example is just changing the secret "example" key:
`{"example": "Test", "number": 42}` -> `{"example": "New Test", "number": 42}`